### DISCUSSION TITLE
Emergent Power: Key Challenges and Capacities for Change Agents of a Second Renaissance - a new report

### DISCUSSION THREAD

**Discussion by catherinet1** [life-itself/community#1060] [Research]
This thread is for reflection on questions and comments regarding Life Itself Research's new report, _Emergent Power: Key Challenges and Capacities for Change Agents of a Second Renaissance_.

The report seeks to respond to the question: **"what inner capacities do change agents in the emerging ecosystem need to develop so that the ecosystem becomes more powerful and better able to contribute to life-serving socio-ecological transformation?"**

You can read the Executive Summary and/or the full report here: https://lifeitself.org/blog/emergent-power-report-2024.

We welcome feedback, comments, questions, reflections... 🙏

**Comment by martinprogrammer** [comment]
@catherinet1  Great and comprehensive report - there's so much stuff there that literally jumped at me and helped with filling in the missing bits.  

I also have a few reflections and would like to spark some conversation about these, if that's what people are interested in.

**"What is needed to support the ecosystem we see emerging to become a system with real power and influence?"**

I really like that the words power and influence are used, but would like more research (clarification) into defining the power and influence in this context and the analysis of the challenges and effects of this power transition or the change in power dynamics.

I've got this intuitive feeling that there are some uncomfortable concepts that are perceived to have non-tolerant or close-minded connotations or ideological foundations - power being one of them.

As the saying goes - power is never given, it's always taken, so how comfortable are we with the word power?

**Comment by martinprogrammer** [comment]
@catherinet1  **"Despite current ads and slogans, the world doesn’t change one person at a time.
It changes as networks of relationships form among people who discover they
share a common cause and vision of what’s possible. [...] As networks grow and
transform into active, working communities of practice, we discover how life
truly changes, which is through emergence."**

I think that meaningful relationships between people that give birth to the world-changing networks depend on the "key inner capacities" - so, I'd argue that the world does change one person at a time.  

This actually cuts much deeper and is present throughout the report - do we believe that the collective has a capacity for a change?  I think that the only agent for a self-realisation, development of capacities and the change is an individual who then uses power and influence to pass onto another individual (meaningful connections).  Only at that point the capacity is transferred to a collective - that indeed has power and influence to grow further.
